archinstall/lib/disk.py

```diff
@@ -539,11 +541,9 @@ def add_partition(self, partition_type, start, end, partition_format=None):
 
 		if partitioning:
 			start_wait = time.time()
-			while previous_partitions == self.blockdevice.partitions:
-				time.sleep(0.025)  # Let the new partition come up in the kernel
-				if time.time() - start_wait > 10:
-					raise DiskError(f"New partition never showed up after adding new partition on {self} (timeout 10 seconds).")
-
+			time.sleep(0.025)  # Let the new partition come up in the kernel
+			if time.time() - start_wait > 20:
+				raise DiskError(f"New partition never showed up after adding new partition on {self} (timeout 10 seconds).")
```

Comment:
Just a note to myself: We needed the while loop to compare partition layouts, but there's a bug where the length of the lists were identical so this while loop would just fly by anyway. The new #426 will hopefully handle this more gracefully. So merging this in anyway.